### PR TITLE
Cancel the current button action

### PR DESCRIPTION
**Pull request by polco**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -294,3 +294,12 @@ function buttonBehavior(button, options) {
 }
 
 module.exports = buttonBehavior;
+
+/**
+ * Cancel the tap on the current button
+ */
+buttonBehavior.cancel = function () {
+	if (current) {
+		current.emit('tapend', true);
```
Comment (ronkorving) [inline index.js]: Should this not unset current? (*not* a rhetorical question)

Comment (polco) [inline index.js]: it's done by the callback method of the tapend event

File: index.js [hunk 2]
```diff
@@ -294,3 +294,12 @@ function buttonBehavior(button, options) {
 }
 
 module.exports = buttonBehavior;
+
+/**
+ * Cancel the tap on the current button
+ */
+buttonBehavior.cancel = function () {
```
Comment (ronkorving) [inline index.js]: Why put this all the way at the bottom of the file? Is there no more logical place?

File: index.js (hunk 1 above)
Comment (ronkorving) [inline index.js]: Understood.

File: index.js (hunk 2 above)
Comment (ronkorving) [inline index.js]: You explained in person, and you're right. I stand corrected :)

**Comment by ronkorving** [comment]
:+1: :ship: :shipit:

**Pull request merged**